tests/test_web_log.py

```diff
@@ -43,13 +43,42 @@ def test_access_logger_format() -> None:
     Ref: https://stackoverflow.com/a/46102240/595220
     """,  # noqa: E501
 )
-def test_access_logger_atoms(mocker) -> None:
-    utcnow = datetime.datetime(1843, 1, 1, 0, 30)
-    mock_datetime = mocker.patch("aiohttp.datetime.datetime")
-    mock_getpid = mocker.patch("os.getpid")
-    mock_datetime.utcnow.return_value = utcnow
-    mock_getpid.return_value = 42
-    log_format = '%a %t %P %r %s %b %T %Tf %D "%{H1}i" "%{H2}i"'
+@pytest.fixture
```

Comment:
I'm pretty sure that skip marker doesn't work on fixtures.

Reply:
but if use parameterizing, it seems to have to use fixtrue

Reply:
Yes, but I'm talking about that the marked is supposed to skip things depending on the condition (PyPy). And if you apply it to fixtures, not test functions it just doesn't have any effect meaning it will never skip any tests.

Reply:
ok. I need to check some documents.

Reply:
Currently, it's okay since you removed the fixture.

Reply:
but...